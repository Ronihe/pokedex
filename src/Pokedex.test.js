import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Pokedex from './Pokedex';

it('renders without crashing', function() {
  shallow(<Pokedex />);
});

it('matches snapshot for adopted dogs', function() {
  let wrapper = shallow(<Pokedex />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
