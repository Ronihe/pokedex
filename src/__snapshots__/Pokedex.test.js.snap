// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for adopted dogs 1`] = `
<ul>
  <span>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/4.png"
        name="Charmander"
        type="fire"
      />
    </li>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/7.png"
        name="Squirtle"
        type="water"
      />
    </li>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/11.png"
        name="Metapod"
        type="bug"
      />
    </li>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/12.png"
        name="Butterfree"
        type="flying"
      />
    </li>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/25.png"
        name="Pikachu"
        type="electric"
      />
    </li>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/39.png"
        name="Jigglypuff"
        type="normal"
      />
    </li>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/94.png"
        name="Gengar"
        type="poison"
      />
    </li>
    <li>
      <Pokecard
        image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/133.png"
        name="Eevee"
        type="normal"
      />
    </li>
  </span>
</ul>
`;
